### PR TITLE
PR:  Algo quick sort in Java

### PR DESCRIPTION
**Pull request by Sanchayan12**
This is a PR for quick sort issue in java

### CODE REVIEW

**Review by Abishethvarman — APPROVED** [review]
Approved to merge

File: 1-data-structures/1-java/4-quick sort [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Quick sort is a popular sorting algorithm in computer science that uses a divide-and-conquer approach to sort a list of elements. It works by selecting a "pivot" element from the list and then partitioning the other elements into two sub-arrays according to whether they are less than or greater than the pivot. This process is then applied recursively to the sub-arrays until the entire list is sorted.
```
Comment (Thevapriyan07) [inline 1-data-structures/1-java/4-quick sort:1]: This file not like readme file you have to change the file name with Quick-sort.MD

**Review by Thevapriyan07 — CHANGES_REQUESTED** [review]
change the file name

File: 1-data-structures/1-java/4-quick sort (hunk 1 above)
Comment (Sanchayan12) [reply 1-data-structures/1-java/4-quick sort:1]: I have made the change

**Review by Thevapriyan07 — APPROVED** [review]

**Pull request merged**